jsonnet/kube-prometheus/components/prometheus.libsonnet

```diff
@@ -276,6 +277,7 @@ function(params) {
         labels: p._config.commonLabels,
       },
       externalLabels: p._config.externalLabels,
+      enableFeatures: p._config.enableFeatures,
```

Comment:
We cannot expose all the features of prometheus-operator (yet) and I wonder if we should expose this one in an easier way. The flag is meant for the more experienced users as everything behind it [is considered experimental](https://prometheus.io/blog/2021/02/17/introducing-feature-flags/) and it still can be set by simply adding:

```jsonnet
prometheus+: {
  prometheus+: {
    spec+: {
      enableFeatures: []
    }
  }
}
```

@prometheus-operator/kube-prometheus-reviewers any ideas? :)

Reply:
I am not very fluent with jsonnet so it didn't even came to my mind that I can do what you suggested.
```
prometheus+: {
  prometheus+: {
    spec+: {
      enableFeatures: []
    }
  }
}
```

This serves my use-case, so I'll leave upto you for what to do with this PR. Thanks!